ibm-cloud-architecture/Learning-Kabanero-101
# More doc updates and some common files added.

**Issue**
Wanted to sure up the docs a little bit more and create common templates for some of the documentation.

**What I did**
Created common documentation files in oc_routes1.adoc and kabanero42.adoc. 

**How to test**
Pull and build, Make sure it builds and docs justify the kabanero 101 experience.
